build.cmd

```diff
@@ -19,6 +19,9 @@ IF NOT EXIST build.fsx (
 SET BuildTarget=
 if "%BuildRunner%" == "MyGet" (
   SET BuildTarget=NuGet
+
```

Comment:
It's pretty simple - anything after `echo` up to `>` is treated as a string, and is placed into RELEASE_NOTES.md, overwriting it. Then, the same follows, but rather appends an indented description. All it does is overwrite the existing RELEASE_NOTES.md with

```
### <whatever version the package is at
	* git build
```